app/scripts/services/SongPlayer.js

```diff
@@ -60,6 +60,13 @@
         * @type {Object}
         */ 
         SongPlayer.currentSong = null;
+        
+        /**
+        * @desc volume control
+        * @type {Number}
+        */ 
+        SongPlayer.volume = 80;
```

Comment:
Does this ever change?   (Try adding `console.log`s to check if you aren't sure).  Where does it change?